src/lib/isMobilePhone.js

```diff
@@ -97,6 +97,7 @@ const phones = {
   'fr-FR': /^(\+?33|0)[67]\d{8}$/,
   'fr-GF': /^(\+?594|0|00594)[67]\d{8}$/,
   'fr-GP': /^(\+?590|0|00590)[67]\d{8}$/,
+  'fr-HT': /^(\+509|00509)[3|4]\d{7}$/,
```

Comment:
Why is this not;
```suggestion
  'fr-HT': /^(\+?509|0|00509)[3|4]\d{7}$/,
```
There can be a valid reason for this of course, I'm just wondering.

Reply:
Thank you for your comment and suggestion. For the international country code, you must use the '+' symbol (or in its defect replace it with '00' ), and after then the international country code, so '+' symbol can't be optional without replace it.